reviews/graduation-coredns.md

```diff
@@ -0,0 +1,51 @@
+
+# CoreDNS Graduation Application
+
+CoreDNS joined CNCF in March 2017 as inception project and graduated as incubation project on February 2018
+
+It has grown significantly over time either as a DNS Server available in Cloud Native environment, either as a key component of Kubernetes project:
+- In Fall 2017, CoreDNS was proposed to become a DNS Discovery Service for Kubernetes.
+- First version as alpha feature was released in v1.9 of Kubernetes - Dec 2017
+- CoreDNS is considered the “default” DNS Discovery Service for Kubernetes v1.13 - to be released in Dec 2018
+
+The following application links to the required information to become a graduated project.
+
+## CoreDNS fulfills all the incubating and graduation criteria:
+
+### Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope.
+
+* Bose, SoundCloud, Infoblox, Zalando, Tradeshift, Yandex, AdGuard are knows users of CoreDNS. More adopters are listed [here](https://github.com/coredns/coredns/blob/master/ADOPTERS.md).
```

Comment:
More adopters and details of their use

Reply:
let me think how I can get some information here...